dataeng/resources/snowflake-validate-stitch.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -ex
+
+# Calculate the start of the validation window as 10 days prior to the end of the window.
+BEGIN_TIME=$(date -d "${SQOOP_START_TIME} - 10 days")
```

Comment:
nit: maybe call it COMPARISON_START_TIME ?  You could even define a COMPARISON_END_TIME too, and include that in the (already useful) comment.